src/corehost/cli/nethost/nethost.cpp

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "nethost.h"
+#include <error_codes.h>
+#include <fxr_resolver.h>
+#include <pal.h>
+#include <utils.h>
+
+NETHOST_API int NETHOST_CALLTYPE nethost_get_hostfxr_path(
+    char_t * result_buffer,
+    size_t buffer_size,
+    size_t * out_buffer_required_size,
+    const char_t * assembly_path)
+{
+    if (out_buffer_required_size == nullptr)
```

Comment:
Call `trace::setup()` as the first thing, to make sure that tracing works correctly.

Reply:
Also - the code should not write anything to `stderr` which it does by default for errors. See IJW host which does the same thing here:
https://github.com/dotnet/core-setup/blob/7d83e1fa27bb2ef3e3623f37370023446a3d3dbe/src/corehost/cli/ijwhost/ijwthunk.cpp#L118